post_installation_configuration/vsphere-failure-domain-configuration.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: ASSEMBLY
+:context: post-install-vsphere-failure-domain-configuration
+[id="post-install-vsphere-failure-domain-configuration"]
+=  Multiple failure domain configuration for a cluster on vSphere
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+As an administrator, you can specify multiple failure domains for your {product-title} cluster that runs on a VMware VSphere instance. This means that you can distribute key control planes and workload elements among varied hardware resources for a datacenter.
+
+A failure domain describes a unique topology. A failure domain configuration lists parameters that create a topology. The following list states some of these parameters:
+
+* `computeCluster`
+
+* `datacenter`
+
+* `datastore`
+
+* `port-group`
```

Comment:
Where this `port-group` come from? When explaining failureDomains.topology, I did not see it.
```
sh-4.4$  ./openshift-install explain installconfig.platform.vsphere.failureDomains.topology
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <object>
  Topology describes a given failure domain using vSphere constructs

FIELDS:
    computeCluster <string> -required-
      computeCluster as the failure domain This is required to be a path

    datacenter <string> -required-
      datacenter is the vCenter datacenter in which virtual machines will be located and defined as the failure domain.

    datastore <string> -required-
      datastore is the name or inventory path of the datastore in which the virtual machine is created/located.

    folder <string>
      folder is the inventory path of the folder in which the virtual machine is created/located.

    networks <[]string>
      networks is the list of networks within this failure domain

    resourcePool <string>
      resourcePool is the absolute path of the resource pool where virtual machines will be created. The absolute path is of the form /<datacenter>/host/<cluster>/Resources/<resourcepool>.
```

Reply:
Thanks, @sgaoshang . Good point. 

@rvanderp3 , I have an entry for the following entity:

|`port` | The TCP port of the vCenter server.

However, does this entity differ from `port-group`?

Reply:
i think the confusion is 'networks' vs 'portgroup'.  we could probably just update that bullet point to state `network` rather than `port-group`.

Reply:
Thanks, @rvanderp3 . I updated the list to state `networks`. 

@sgaoshang . Is this change OK with you?